redhat-developer/developers.redhat.com
# [DO NOT MERGE] [MWES-2846] Update Drupal to RHEL 7 OS and SCL dependencies

**DO NOT MERGE THIS PR**

This PR updates our Drupal image to a RHEL 7 based build that we are going to be using on Managed Platform. In addition to using RHEL 7 it uses SCL for all dependencies e.g.

* PHP
* NodeJS
* Ruby
* Git

Apache is updated to 2.4 and configured to log to STDOUT which means we can deprecate and remove logrotate. Apache also runs as a non-root user `apache` in group `root` and listens now on port `8080`. I've updated all `docker-compose.yml` files to correctly bind the new port to the expected address.

### JIRA Issue Link
* https://issues.jboss.org/browse/MWES-2846

### Verification Process

* The build should go green
* Check Drupal and ensure that there are no straight looking errors in the log files
* Check that there is no hanging Drupal configuration to import
